src/ts/uimanager.ts

```diff
@@ -50,6 +50,10 @@ import {Spacer} from './components/spacer';
 import {UIUtils} from './uiutils';
 import {ArrayUtils} from './arrayutils';
 import {BrowserUtils} from './browserutils';
+import i18n from './i18n';
+
+// Interfaces and types
+import { Panel } from './translations/translation-types';
```

Comment:
All imports should be in a single block at the top of the file... else IntelliJ will do some weird stuff on reformatting